src/system.rs

```diff
@@ -553,12 +553,12 @@ impl vr::IVRSystem023_Interface for System {
                 // itself doesn't appear to be that important.
                 vr::ETrackedDeviceProperty::SerialNumber_String
                 | vr::ETrackedDeviceProperty::ManufacturerName_String
-                | vr::ETrackedDeviceProperty::ControllerType_String => Some(c"<unknown>"),
+                | vr::ETrackedDeviceProperty::ControllerType_String => {
+                    Some(CString::new("<unknown>").unwrap())
+                }
                 _ => None,
             },
-            x => input
-                .device_index_to_hand(x)
-                .and_then(|hand| input.get_controller_string_tracked_property(hand, prop)),
+            _ => input.get_device_string_tracked_property(device_index, prop),
```

Comment:
The above `let Some(input) = self.input.get() else ...` should be removed, then:
```suggestion
            _ => self.input.get().and_then(|input| input.get_device_string_tracked_property(device_index, prop)),
```